### PR TITLE
Add option to turn off BIGTIFF

### PR DESCRIPTION
**Pull request by banesullivan**
Adds a way to control the COG conversion utility's output and turn off the default BIGTIFF option

### CODE REVIEW

File: utilities/converter/large_image_converter/__init__.py [hunk 1]
```diff
@@ -103,15 +104,20 @@ def _generate_geotiff(inputPath, outputPath, **kwargs):
...
         'yes': 'YES',
     }
+    predictor = {
+        'horizontal': 'STANDARD',
+        'float': 'FLOATING_POINT',
+    }.update(yes_no)
     options.update({k: v for k, v in kwargs.items() if v not in (None, '')})
-    cmdopt = ['-of', 'COG', '-co', 'BIGTIFF=YES']
+    cmdopt = ['-of', 'COG']
+    cmdopt += ['-co', 'BIGTIFF=%d' % predictor[yes_no['bigTiff']]]
```
Comment (manthey) [inline utilities/converter/large_image_converter/__init__.py:120]: I think you want `predictor[yes_no['bigTiff']]` to be `yes_no[options['bigTiff']]`.

**Comment by manthey** [comment]
There is a `IF_NEEDED` value for the `BIGTIFF` option that we should use instead.

**Comment by manthey** [comment]
Actually, the GDAL docs say:
```
  -  IF_NEEDED will only create a BigTIFF if it is clearly needed (in
      the uncompressed case, and image larger than 4GB. So no effect
      when using a compression).
   -  IF_SAFER will create BigTIFF if the resulting file \*might\*
      exceed 4GB. Note: this is only a heuristics that might not always
      work depending on compression ratios.
```
Neither are guaranteed.  Looking at the code, `IF_SAFER` checks if the uncompressed image is > 2GB regardless of compression.  `IF_NEEDED` only if the uncompressed image is > 4GB AND uncompressed.  So `IF_SAFER` will probably mostly be sufficient as the default, and then we can allow the values to be overridden as in this PR.

File: utilities/converter/large_image_converter/__init__.py (hunk 1 above)
Comment (banesullivan) [reply utilities/converter/large_image_converter/__init__.py:120]: Whoops. This is what happens when I try to multitask and not run the code

**Comment by manthey** [comment]
#594 is going to cause this PR to need to be refactored.  It also will switch the default to `IF_SAFER`, which might be sufficient for most use cases.

**Comment by manthey** [comment]
I'm going to close this, as I think the changes in #594 obviated the need for it.  Please reopen if you think it is still necessary.

**Pull request closed without merging**